--- origsrc/parole-0.2.0.6/src/Makefile.am	2011-04-17 06:53:19.000000000 -0500
+++ src/parole-0.2.0.6/src/Makefile.am	2011-06-01 17:39:31.871611200 -0500
@@ -72,6 +72,8 @@ parole_SOURCES =				\
 	parole-plugin-player.c			\
 	parole-plugin-player.h
 
+parole_LDFLAGS = -export-dynamic -Wl,--out-implib,libparole.a
+
 parole_LDADD =					\
 	$(top_builddir)/parole/libparole.la 	\
 	$(top_builddir)/dbus/libparoledbus.la 	\
--- origsrc/parole-0.2.0.2/plugins/properties/Makefile.am	2010-01-25 05:22:07.000000000 -0600
+++ src/parole-0.2.0.2/plugins/properties/Makefile.am	2010-01-28 13:47:14.921197800 -0600
@@ -23,12 +23,16 @@ stream_properties_la_CFLAGS =			\
 	$(LIBXFCE4UTIL_CFLAGS)
 
 stream_properties_la_LIBADD =			\
+	$(GTK_LIBS)				\
+	$(LIBXFCE4UTIL_LIBS)			\
 	$(TAGLIBC_LIBS)
 
 stream_properties_la_LDFLAGS =			\
 	-avoid-version				\
 	-export-dynamic				\
 	-module					\
+	-no-undefined				\
+	-Wl,$(top_builddir)/src/libparole.a	\
 	$(PLATFORM_LDFLAGS)
 
 #
--- origsrc/parole-0.2.0.2/plugins/tray/Makefile.am	2010-01-25 05:22:07.000000000 -0600
+++ src/parole-0.2.0.2/plugins/tray/Makefile.am	2010-01-28 13:48:16.371712500 -0600
@@ -27,9 +27,14 @@ tray_icon_la_LDFLAGS =				\
 	-avoid-version				\
 	-export-dynamic				\
 	-module					\
+	-no-undefined				\
+	-Wl,$(top_builddir)/src/libparole.a	\
 	$(PLATFORM_LDFLAGS)
 
 tray_icon_la_LIBADD =				\
+	$(GTK_LIBS)				\
+	$(LIBXFCE4GUI_LIBS)			\
+	$(LIBXFCE4UTIL_LIBS)			\
 	$(LIBNOTIFY_LIBS)
 
 #
--- origsrc/parole-0.2.0.2/plugins/window-title/Makefile.am	2010-01-25 05:22:07.000000000 -0600
+++ src/parole-0.2.0.2/plugins/window-title/Makefile.am	2010-01-28 13:48:54.892915800 -0600
@@ -21,10 +21,16 @@ window_title_la_CFLAGS =			\
 	$(GTK_CFLAGS)				\
 	$(LIBXFCE4UTIL_CFLAGS)
 
+window_title_la_LIBADD =			\
+	$(GTK_LIBS)				\
+	$(LIBXFCE4UTIL_LIBS)
+
 window_title_la_LDFLAGS =			\
 	-avoid-version				\
 	-export-dynamic				\
 	-module					\
+	-no-undefined				\
+	-Wl,$(top_builddir)/src/libparole.a	\
 	$(PLATFORM_LDFLAGS)
 
 #
